modules/apps/foundation/portal/portal-upload/src/main/java/com/liferay/portal/upload/configuration/UploadServletRequestConfiguration.java

```diff
@@ -29,10 +29,16 @@
 )
 public interface UploadServletRequestConfiguration {
 
-	@Meta.AD(deflt = "104857600", required = false)
+	@Meta.AD(
+		deflt = "104857600", description = "max-size-help",
+		name = "overall-maximum-upload-request-size", required = false
+	)
```

Comment:
Hey @brianchandotcom,

Do we have a rule on the format we should use for the key for name and description? I see both dots and dashes being used in portal. My understanding is that if the key comes from an existing lang key, we keep the dash notation. Otherwise, we use the dot notation.

/cc @jorgeferrer